### ISSUE TITLE
Profiles have extra subtype key

### ISSUE THREAD

**Issue by juliangruber** [p2pcommons/sdk-js#47]
```js
const P2PCommons = require('@p2pcommons/sdk-js')

const main = async () => {
  const p2p = new P2PCommons({ baseDir: `/tmp/${Math.random()}` })
  await p2p.ready()
  const { rawJSON: { url } } = await p2p.init({
    type: 'profile',
    title: 'Patch'
  })
  console.log(await p2p.get(url))
  await p2p.destroy()
}

main().catch(err => {
  console.error(err)
  process.exit(1)
})
```

```bash
$ node test.js
{ rawJSON:
   { title: 'Patch',
     description: '',
     url:
      '462008fa23c2bba4c08059faf67bb00fceff03e773704c5780be9b91a9f60006',
     links: Record$ { license: [Array], spec: [Array] },
     type: 'profile',
     subtype: '',
     main: '',
     follows: [],
     contents: [] },
  metadata:
   { isWritable: true, lastModified: 1574704774324, version: 3 } }
```

Afaik a `type="profile"` shouldn't have a `subtype` key.

**Comment by chartgerink** [comment]
The [module spec](https://github.com/p2pcommons/specs/blob/master/module.md) specifies that it does?

My thinking was that, considering Wikidata includes people as objects, it would be a good idea to at least have the option to include that in the future ([example](https://www.wikidata.org/wiki/Q28922414)). Also, a profile may be used by an entity that isn't a person, so we could also use identifiers for those.

In other words, I don't think this is a bug.

**Comment by juliangruber** [comment]
> The [module spec](https://github.com/p2pcommons/specs/blob/master/module.md) specifies that it does?

Oh, I should have checked!

> My thinking was that, considering Wikidata includes people as objects, it would be a good idea to at least have the option to include that in the future ([example](https://www.wikidata.org/wiki/Q28922414)). Also, a profile may be used by an entity that isn't a person, so we could also use identifiers for those.

Ok my confusion probably came from this not being implemented in the CLI (yet) then :)
